go/vt/worker/restartable_result_reader.go

```diff
@@ -172,7 +185,7 @@ func (r *RestartableResultReader) nextWithRetries() (*sqltypes.Result, error) {
 	defer retryCancel()
 
 	// Note: The first retry will be the second attempt.
-	attempt := 1
+	attempt := 0
```

Comment:
Now the first retry is the second attempt. Unless I misunderstand something about go.